diff --git a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
index bb59351..d966a76 100644
--- a/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
+++ b/node_modules/react-native/ReactCommon/react/renderer/core/graphicsConversions.h
@@ -77,7 +77,9 @@ inline folly::dynamic toDynamic(const YGValue& dimension) {
     case YGUnitPoint:
       return dimension.value;
     case YGUnitPercent:
-      return std::format("{}%", dimension.value);
+      char buffer[256];
+      std::snprintf(buffer, sizeof(buffer), "%.9g%%", dimension.value);
+      return buffer;
   }
 
   return nullptr;
